hud.lua

```diff
@@ -66,7 +66,8 @@ minetest.register_globalstep(function(dtime)
 			hud = {}
 			areas.hud[name] = hud
 			hud.areasId = player:hud_add({
-				hud_elem_type = "text",
+				hud_elem_type = "text", -- ignored on new clients
```

Comment:
this is wrong, clients never see the difference

should be "ignored by new servers"

Reply:
Done in https://github.com/minetest-mods/areas/pull/78/commits/000015c7349927172c078dbe9350b4f0d018d68e